api/PayrollProcessor.Web.Api/PayrollProcessor.Web.Api.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="5.4.1" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="5.4.1" />
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.8.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.7"/>
```

Comment:
System.Text.Json had issues parsing decimal so I brought in NewtonsoftJson.

reference:
stackoverflow.com/questions/57626878/the-json-value-could-not-be-converted-to-system-int32

Reply:
Interesting!